### PR TITLE
Not create rbac roles on k8s cluster that have rbac disabled.

### PR DESCRIPTION
**Pull request by kokhang**
Fixes https://github.com/rook/rook/issues/1221

### CODE REVIEW

**Comment by travisn** [comment]
@kokhang I don't think this PR is needed. Comment added to #1221.

**Comment by travisn** [comment]
we can revisit if #1221 is reopened

**Comment by kokhang** [comment]
Reopen

**Comment by travisn** [comment]
do we need to change this fix at all given the details we learned in this thread? 
https://gist.githubusercontent.com/jbguerraz/cfdf8c13705d76e4e4ed92389f9cd884/raw/5cf0f7681a01f585f934f8af9243325df64edd25/gistfile1.txt

**Comment by jbguerraz** [comment]
@travisn  Indeed it could either (try to) create (then useless) RBAC resources and report API error(s) but continue to run or it could not try at all to create useless RBAC resources at all if one don't plan to use RBAC authz.
I tend to believe it's better to not "poluate" the system with unused resources, so I believe @kokhang 's fix is great as is.

File: pkg/operator/k8sutil/rbac.go [hunk 1]
```diff
@@ -30,13 +31,21 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+const (
+	enableRBACEnv = "ENABLE_RBAC"
```
Comment (travisn) [inline pkg/operator/k8sutil/rbac.go]: what if we called this `RBAC_ENABLED`?

**Pull request merged**